src/pages/billing/List.js

```diff
@@ -438,6 +438,7 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
         [!isLoadingProjects && _.isEmpty(billingProjects), () => h(CreateNewBillingProjectWizard, {
           billingAccounts,
           onSuccess: billingProjectName => {
+            Ajax().Metrics.captureEvent(Events.billingProjectWizardProjectCreated, { billingProject: billingProjectName })
```

Comment:
Ah, here is where we should put `billing:creation:gcpBillingProjectCreated` (no buttonName).